### PR TITLE
Upgrade ember

### PR DESCRIPTION
**Pull request by herzzanu**
Addresses #82, #83 and #84

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,54 +8,58 @@
...
-    "ember-data": "^2.3.0",
-    "ember-disable-prototype-extensions": "^1.1.0",
-    "ember-disable-proxy-controllers": "^1.0.1",
-    "ember-export-application-global": "^1.0.4",
-    "ember-load-initializers": "^0.5.0",
-    "ember-power-calendar": "0.1.4",
-    "ember-resolver": "^2.0.3",
+    "broccoli-asset-rev": "2.4.5",
+    "ember-ajax": "3.0.0",
+    "ember-cli": "~2.17.2",
```
Comment (Exelord) [inline package.json]: 2.18 is out now :D

File: package.json [hunk 2]
```diff
@@ -8,54 +8,58 @@
...
     "bookingsync",
     "bsy"
   ],
   "dependencies": {
-    "ember-cli-babel": "^5.1.5",
-    "ember-one-way-controls": "^0.8.0",
-    "ember-power-select": "^0.10.9",
-    "ember-modal-dialog": "^0.8.4",
-    "ember-cli-htmlbars": "^1.0.8"
+    "ember-cli-babel": "6.6.0",
```
Comment (Exelord) [inline package.json]: hmm, look pretty old. Did u run `ncu` on it?

File: .eslintrc.js [hunk 3]
```diff
@@ -18,5 +21,42 @@ module.exports = {
     'ember/order-in-models': 1,
     'ember/order-in-controllers': 1,
     'ember/alias-model-in-controller': 0
-  }
+  },
+  overrides: [
```
Comment (Exelord) [inline .eslintrc.js]: It's from Ember-cli - Im assuming.

File: .travis.yml [hunk 4]
```diff
@@ -41,6 +41,7 @@ install:
   - yarn install --no-lockfile --non-interactive
 
 script:
+  - yarn lint:js
```
Comment (Exelord) [inline .travis.yml]: Isn't the lint called during the build?

File: .eslintrc.js (hunk 3 above)
Comment (herzzanu) [reply .eslintrc.js]: Yes, that's true.

File: .travis.yml (hunk 4 above)
Comment (herzzanu) [reply .travis.yml]: It's from ember-cli as well.

**Pull request merged**